flask-web/application.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+
+from flask import (Flask, redirect, render_template, request,
+                   send_from_directory)
+
+application = Flask(__name__, static_folder='static')
+
+
+@application.route('/css/<path:path>')
```

Comment:
I believe you can just make this all into one function correct?

```
@application.route('/static/<path:path>')
def send_static(path):
    return send_from_directory('static', path)
```
Unless you think there is more utility in having 5 separate functions?

Reply:
Ah much more efficient - great